features/nanostack/mbed-mesh-api/mbed_lib.json

```diff
@@ -153,6 +153,22 @@
         "wisun-uc-dwell-interval": {
             "help": "Unicast dwell interval. Range: 15-255 milliseconds",
             "value": 0
+        },
+        "certificate-header": {
+            "help": "Certificate header",
```

Comment:
What is the type or range of the `value` field? Please document that as well.

Others have already documentation.

Reply:
Corrected